src/components/complex/CreateProposalModal/index.jsx

```diff
@@ -117,11 +117,11 @@ const CreateProposalModal = ({ show, onClose }) => {
         const preset = presets[selectedPreset]
         const action = await preset.prepare()
         if (action) {
-          setScript(encodeCallScript([action]))
+          setScript(encodeCallScript(action))
```

Comment:
Does this change work with the already existing presets?

Reply:
Yes, I added https://github.com/pnetwork-association/dao-v2-ui/blob/30903bd9bad30b8c2dbf267581b788e269533270/src/utils/vote-presets.js#L120 for `paymentFromTreasury` and custom now need to pass an array with an object inside and not just an object.

Reply:
Better use `actions` instead, so the array can be guessed easily